### DISCUSSION TITLE
Question about working with template files with PstFrom

### DISCUSSION THREAD

**Discussion by BojanMilinic** [gmdsi/GMDSI_notebooks#145] [Q&A]
When working with PstFrom, the add_parameters() method will write a template file for PEST. Can you set it up where you can add a template file that was already created? Does that even make sense when using PstFrom?

**Comment by mnfienen** [comment]
Hey @BojanMilinic  - yes, you can mix and match once you have created the PST object from PstFrom. At that point, you can use `pst.add_parameters(<tpl_file>)` using your custom template file.
